controllers/new.go

```diff
@@ -217,6 +223,40 @@ func (k *kubeResources) newPolicyEngineService(instance *cv1.UpdateService) *cor
 	}
 }
 
+func (k *kubeResources) newMetadataService(instance *cv1.UpdateService) *corev1.Service {
+	name := nameMetadataService(instance)
+	return &corev1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: instance.Namespace,
+			Labels: map[string]string{
+				"app": name,
```

Comment:
nit: I would expect the `app` label to be the same as the route that exposes the service? but I haven't looked too deep in how do we organize OSUS resources :shrug:

Reply:
`newMetadataService` is using `nameMetadataService` for `app` and its own `name`.  This pattern-matches the existing `newPolicyEngineService`

`newMetadataRoute` is using `nameMetadataRoute` for its own `name`, and `nameDeployment` for `app`.  This pattern-matches the existing `newPolicyEngineRoute`.

There are some existing patterns that don't make sense to me, like why the Services expose the status ports that I'd expect only the kubelet to need access to (and the kubelet gets at the containers without passing through the Service).  But I've left that kind of refactoring to follow-up work and just matched existing patterns for this new feature.